cookbook/controller/error_pages.rst

```diff
@@ -214,11 +214,94 @@ will be passed two parameters:
     A :class:`\\Symfony\\Component\\HttpKernel\\Log\\DebugLoggerInterface`
     instance which may be ``null`` in some circumstances.
 
+Extending from the Default ExceptionController Class
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Instead of creating a new exception controller from scratch you can, of course,
 also extend the default :class:`Symfony\\Bundle\\TwigBundle\\Controller\\ExceptionController`.
 In that case, you might want to override one or both of the ``showAction()`` and
 ``findTemplate()`` methods. The latter one locates the template to be used.
 
+To create your own controller logic extending from the ExceptionController, simply
+create a controller with the specified method you want to override. In this
+example, we gonna override the showAction method::
```

Comment:
Could we explain very briefly why we want to override the method? Instead of:

```
we gonna override the showAction method::
```

you can say something like:

```
the application overrides the ``showAction()`` method to do THIS and THAT::
```